docker/Dockerfile.ci_cpu

```diff
@@ -119,6 +119,11 @@ RUN bash /install/ubuntu_install_androidsdk.sh
 ENV ANDROID_HOME=/opt/android-sdk-linux/
 ENV ANDROID_NDK_HOME=/opt/android-sdk-linux/ndk/21.3.6528147/
 
+# Install CMSIS_NN
```

Comment:
why install into ci_cpu?

Reply:
CI_CPU already had this because it has `install/ubuntu_install_ethosu_driver_stack.sh`